app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,9 @@
+class MerchantsController < ApplicationController
+  def index
+    @facade = MerchantsFacade.new
+  end
+
+  def show
+    @facade = MerchantsFacade.new(params[:id])
+  end
```

Comment:
let's call these instance variables something other than @ facade. They could be @ merchants or @ merchant.